src/test/java/com/slcube/shop/business/category/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package com.slcube.shop.business.category.service;
+
+import com.slcube.shop.business.category.dto.CategoryResponseDto;
+import com.slcube.shop.business.category.dto.CategorySaveRequestDto;
+import com.slcube.shop.business.category.dto.CategoryUpdateRequestDto;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.Transactional;
+
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+@Transactional
+class CategoryServiceTest {
+
+    @Autowired
+    private CategoryService categoryService;
+
+    @Test
+    @DisplayName("카테고리 저장")
+    void saveCategory() {
+        // given
```

Comment:
```suggestion
    void saveCategoryTest() {
        // given
```
주로 컨벤션으로 인해서 method 뒤에 `Test` 붙입니다!